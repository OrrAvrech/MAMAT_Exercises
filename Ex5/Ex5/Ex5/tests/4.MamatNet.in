New a a
New b b
New c c
Login Admin 1234
Messages
New a
New b
New c
Back
Messages
Open 3
Write test
Back
Open 3
Write x^2+y^2=1
Back
Back
Logout
Login a a
Messages
New Admin
Back
Messages
New b
Back
Messages
Open 3
Write :)
Back
Back
Logout
Login Admin 1234
Messages
Back
Delete a
Search a
Messages
Back
Logout
Login b b
Messages
Open 2
Back
Open 2
Write abcdweji
Back
Back
Logout
New x x
Login x x
Logout
Login Admin 1234
Messages
Open 2
Write :)
Back
Open 2
Back
Back
Delete x
Delete a
Delete b
Delete c
Messages
Delete 1
Back
Messages
Delete 1
Delete 1
Delete 1
Delete 1
Back
Logout
Exit
