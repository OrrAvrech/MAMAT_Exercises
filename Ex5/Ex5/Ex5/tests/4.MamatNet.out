Welcome to MamatChat.
Hi Admin, you have no new messages.
No conversations
Hi Admin, you have no new messages.
Conversations:
1) Participants: Admin, c
2) Participants: Admin, b
3) Participants: Admin, a
Participants: Admin (read), a (read)
Conversations:
1) Participants: Admin, a
2) Participants: Admin, c
3) Participants: Admin, b
Participants: Admin (read), b (read)
Conversations:
1) Participants: Admin, b
2) Participants: Admin, a
3) Participants: Admin, c
Hi Admin, you have no new messages.
Welcome to MamatChat.
Hi a, you have new messages.
Conversations:
1) (UNREAD) Participants: Admin, a
Hi a, you have new messages.
Conversations:
1) Participants: Admin, a
2) (UNREAD) Participants: Admin, a
Hi a, you have new messages.
Conversations:
1) Participants: a, b
2) Participants: Admin, a
3) (UNREAD) Participants: Admin, a
Participants: Admin (read), a (read)
Admin : test
Conversations:
1) Participants: Admin, a
2) Participants: a, b
3) Participants: Admin, a
Hi a, you have no new messages.
Welcome to MamatChat.
Hi Admin, you have new messages.
Conversations:
1) (UNREAD) Participants: Admin, a
2) Participants: Admin, a
3) Participants: Admin, b
4) Participants: Admin, c
Hi Admin, you have new messages.
No user matches the pattern
Conversations:
1) (UNREAD) Participants: Admin
2) Participants: Admin
3) Participants: Admin, b
4) Participants: Admin, c
Hi Admin, you have new messages.
Welcome to MamatChat.
Hi b, you have new messages.
Conversations:
1) Participants: b
2) (UNREAD) Participants: Admin, b
Participants: Admin (read), b (read)
Admin : x^2+y^2=1
Conversations:
1) Participants: b
2) Participants: Admin, b
Participants: Admin (read), b (read)
Admin : x^2+y^2=1
Conversations:
1) Participants: Admin, b
2) Participants: b
Hi b, you have no new messages.
Welcome to MamatChat.
Hi x, you have no new messages.
Welcome to MamatChat.
Hi Admin, you have new messages.
Conversations:
1) (UNREAD) Participants: Admin, b
2) (UNREAD) Participants: Admin
3) Participants: Admin
4) Participants: Admin, c
Participants: Admin (read)
Admin : test
a : :)
Conversations:
1) Participants: Admin
2) (UNREAD) Participants: Admin, b
3) Participants: Admin
4) Participants: Admin, c
Participants: Admin (read), b (read)
Admin : x^2+y^2=1
b : abcdweji
Conversations:
1) Participants: Admin
2) Participants: Admin, b
3) Participants: Admin
4) Participants: Admin, c
Hi Admin, you have no new messages.
User doesn't exist
Conversations:
1) Participants: Admin
2) Participants: Admin
3) Participants: Admin
4) Participants: Admin
Hi Admin, you have no new messages.
Conversations:
1) Participants: Admin
2) Participants: Admin
3) Participants: Admin
Invalid conversation number
Hi Admin, you have no new messages.
Welcome to MamatChat.
